sklearn/linear_model/ridge.py

```diff
@@ -1090,7 +1096,7 @@ def __init__(self, alphas=(0.1, 1.0, 10.0),
                  fit_intercept=True, normalize=False, scoring=None,
                  cv=None, gcv_mode=None,
                  store_cv_values=False):
-        self.alphas = alphas
+        self.alphas = np.asarray(alphas)
```

Comment:
Why doing so?

Reply:
I was suggested to make that changes a few lines above: to add a conversion in the `__init__` and then remove the float. It's done in the __init__ of `_RidgeGCV`.